### DISCUSSION TITLE
understanding MeteoFrance API

### DISCUSSION THREAD

**Discussion by farfromrefug** [open-meteo/open-meteo#319] [Q&A]
Right now in my app i am using meteo france api through `https://webservice.meteofrance.com/forecast` and others endpoints (rain, observation/gridded, warning/full).
The dream would be to have open-meteo to return the same quality of data as meteofrance so that i can switch fully to open-meteo.
A few things question me:
* there are a lot of fields available in meteofrance which are with open-meteo-meteofrance: `rain snow limit`, `iso0`, `probability_forecast`, ... Is that not available through the open meteofrance data you are fetching?
* if i compare the result from your `open-meteo-meteofrance` to `https://webservice.meteofrance.com/forecast` i see very different results. Temperatures are a bit different but not too much (rounding?). Wind speed,direction, uvindex though are very different which raises the question is that because they compute it differently from their data?

**Comment by patrick-zippenfenig** [comment]
Is there public documentation available for the Meteofrance forecast api? I struggle to find anything useful.

I am using most of the open-data Meteofrance models and retrieve them via AWS: https://mf-models-on-aws.org/en/doc/models/arome-france/. The open-data available from Meteofrance is severely limited compared to what they actually calculate. Also access to other models like ensembles is not possible via open-data

Many difference could be the result of different model update times and resolutions. Some variables like UV index are not available from meteofrance and are instead used from the American GFS model. The [Open-Meteo MeteoFrance API](https://open-meteo.com/en/docs/meteofrance-api) contains more details of all weather models and variables. However the overall forecast should be similar. Did you notice any larger differences?
